app/views/links/listing_property.erb

```diff
@@ -0,0 +1,17 @@
+<h2> Property Information </h2>
+
+<form action='/property' method='post'>
+  <label for='name'>
+    Name: <input name='name' type='text'>
```

Comment:
Change property name to title to avoid confusion with name for property and user.